data_setup/download_data.py

```diff
@@ -0,0 +1,4 @@
+import gdown
+
+link = "https://drive.google.com/drive/folders/1IwJjwSKpXYWWALKXM8nghqmlSvyu3BYq"
+gdown.download_folder(link, quiet=True)
```

Comment:
Does it also fail quietly?

Reply:
removed this flag, outputs error if occurs one (eg no network connection)